src/App.fsproj

```diff
@@ -1,17 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net9.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
by default ubuntu images on github has NET8 so best to stick with that for now, has likely no major impact on fable compilation